jdat_notebooks/README.md

```diff
@@ -140,21 +142,21 @@ follow-on commit, with implementation of all comments then being the step that a
 ready to be called "Integrated".
 
 Once the notebook authors (original author and developer/reviewer) have agreed it is ready, one of them follows 
-the Pull Request workflow as described above, but with the notebook title now changed to be just the title itself 
-(no "Concept:" or Draft:"). The Pull Request is then reviewed by one of the project scientists, and merged when 
+the Pull Request workflow as described above, but with the notebook title now changed to be just 
+the title itself (no "Draft:" or Baseline:"). The Pull Request is then reviewed by one of the project scientists, and merged when 
```

Comment:
I hadn't thought about this until now, but perhaps we want to replicate the sci/tech review process from the above at this time?  As it stands it's now just "the project scientist", which seems a bit of a bottleneck relative to the baseline review process.

(Of course the sci/tech review could *be* a project scientist, but we don't have to mandate it.)

Reply:
Right. Better say "by a developer and a scientist".
Thank you for catching these things!